file-formats/intm/type/zif_aff_intm_v1.intf.abap

```diff
@@ -0,0 +1,120 @@
+INTERFACE zif_aff_intm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Intelligent Scenario Model</p>
+    "! This is an intelligent scenario model
+    BEGIN OF ty_intelligent_model,
+      "! <p class="shorttext">Model Name</p>
+      "! This is the model name
+      name                  TYPE c LENGTH 30,
+      "! <p class="shorttext">Parent</p>
+      "! This is the parent of the model
+      parent                TYPE c LENGTH 36,
```

Comment:
As far as I understood, this is basically the link to the scenario. Wouldn't it make sense to call this "intelligent scenario" or similar?

Reply:
I've introduced Intelligent Scenario field in the ADT UI for display purpose, but this parent field is hidden as this can be generated only in the Original system and should be transported. I've renamed this field as intelligent_scenario_id.